frontends/bmd/src/pages/unlock_admin_screen.test.tsx

```diff
@@ -51,3 +51,27 @@ test('If passcode is incorrect, error message is shown', () => {
   render(<UnlockAdminScreen auth={auth} />);
   screen.getByText('Invalid code. Please try again.');
 });
+
+test('Hides focus outlines when necessary', () => {
+  const auth = Inserted.fakeCheckingPasscodeAuth();
+  const hideFocusOutlinesCssClass = '.hide-focus-outlines';
+
+  let renderResult = render(
+    <UnlockAdminScreen areFocusOutlinesVisible={false} auth={auth} />
+  );
+  expect(document.querySelector(hideFocusOutlinesCssClass)).toBeInTheDocument();
```

Comment:
This was a tricky thing to test from a purely user-oriented perspective since we're not changing whether or not something is focused, just the styling.

Given the constraint that we have to test styling, I could've alternatively checked the value of the `outline` style, but this isn't possible without somehow rendering the component with the styles from `App.css` applied, also tricky